webpack/base.config.js

```diff
@@ -0,0 +1,39 @@
+import webpack from 'webpack';
+import yargs from 'yargs';
+
+export const argv = yargs
+  .alias('p', 'optimize-minimize')
+  .alias('d', 'debug')
+  .option('port', {
+    default: '8080',
+    type: 'string'
+  })
+  .argv;
+
+const baseConfig = {
+  entry: undefined,
+
+  output: undefined,
+
+  externals: undefined,
+
+  module: {
+    loaders: [
+      { test: /\.js/, loader: 'babel', exclude: /node_modules/ }
+    ]
+  },
+
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        'NODE_ENV': JSON.stringify(argv.optimizeMinimize ? 'production' : 'development')
+      }
+    })
+  ]
+};
+
+if (argv.optimizeMinimize) {
+  baseConfig.devtool = 'source-map';
+}
```

Comment:
`devtool` for tests needs to be `eval` or else failure stack traces don't work. @taion Is proposing that this devtool should be `eval-source-map` and I'm inclined to agree.